views/courses/course.hbs

```diff
@@ -53,38 +53,38 @@
     </div>
     <section class="section-course">
       <div class="course-card">
-        <div class="tabContainer">
+        <div class="tabContainer" active-tab={{activeTab}}>
```

Comment:
auch hier fehlt data-

Reply:
fixed in https://github.com/schul-cloud/schulcloud-client/pull/1235

Reply:
warum ist es da mit drin und nicht hier?